source/components/molecules/PersonField/PersonField.tsx

```diff
@@ -0,0 +1,240 @@
+  padding-bottom: 20px;
+  padding-top: 16px;
+  padding-left: 16px;
+  padding-right: 16px;
+  background: ${props => props.theme.colors.complementary.blue[3]};
+`;
+
+const PersonFieldHeader = styled.View`
+  flex-direction: row;
+`
+const PersonFieldDetails = styled.View`
+  margin-left: 20px;
+  margin-top: 6px;
+`
+
+const PersonFieldInfoName = styled(Text)`
+  font-weight: ${props => props.theme.fontWeights[1]};
+  font-size: ${props => props.theme.fontSizes[4]};
+`
+const PersonFieldInfoRelation = styled(Text)`
+  font-weight: ${props => props.theme.fontWeights[1]};
+  color: ${props => props.theme.colors.primary.blue[1]};
+`
+const PersonFieldInfoPNO = styled(Text)`
+margin-top: 6px;
+  font-size: ${props => props.theme.fontSizes[3]};
+`
+const PersonFieldDivider = styled.View`
+  height: 2px;
+  margin: 0 -16px;
+  margin-top: 20px;
+  background: ${props => props.theme.colors.complementary.blue[1]};
+`;
+
+const PersonFieldFooter = styled.View`
+  margin-top: 16px;
+`;
+const PersonFieldDeleteButton = styled(Button).attrs({
+  z: 0,
+  size: "small"
+})`
+  min-width: 100%;
+  min-height: auto;
+`;
+
+
+const PersonFieldBody = styled.View`
+  padding-top: 12px;
+  padding-bottom: 20px;
+  padding-right: 16px;
+  height: auto;
+`;
+
+const PersonFieldInputContainer = styled.View<{editable: boolean; colorSchema: string;}>`
+font-size: ${props => props.theme.fontSizes[4]}px;
+  flex-direction: row;
+  height: auto;
+  background-color: transparent;
+  border-radius: 4.5px;
+  margin-bottom: 10px;
+  ${props =>
+    props.editable
+      && `
+      background-color: ${props.theme.colors.complementary[props.colorSchema][2]};
+      padding: 10px;
+      `
+  };
+`;
+
+
+const PersonFieldInputLabelWrapper = styled.View`
+  flex: 4;
+  justify-content: center;
+`;
+
+const PersonFieldInputLabel = styled.Text`
+  padding: 4px;
+  font-weight: ${props => props.theme.fontWeights[1]};
+  color: ${props => props.theme.colors.neutrals[1]};
+`;
+
+const PersonFieldInputWrapper = styled.View`
+  flex-direction: column;
+  align-items: flex-end;
+  justify-content: center;
+  flex: 6;
+`;
+
+// eslint-disable-next-line prettier/prettier
+const PersonFieldInput = styled.TextInput`
+  text-align: right;
+  min-width: 70%;
+  font-size: ${props => props.theme.fontSizes[3]}px;
+  font-weight: bold;
+`;
+
+interface Input {
+  label: string,
+  key: string,
+  type: 'text',
+  value: string,
+}
+interface Props {
+  firstName: string;
+  lastName: string;
+  middleName: string;
+  personalNumber: string;
+  relation: string;
+  isEditable: boolean;
+  inputs: Input[];
+  onDelete: () => void;
+  colorSchema: string;
+}
+function PersonField({firstName, lastName, isEditable, personalNumber, relation, colorSchema, inputs, onDelete}: Props)
+{
+  const theme = useContext(ThemeContext);
+  // Check if passed color is defined in theme
+  const validColorSchema = Object.keys(theme.colors.primary).includes(colorSchema)
+    ? colorSchema
+    : 'blue';
```

Comment:
This check is now a helper, getValidColorSchema, defined in theme.js, so that can be used instead.